.travis.yml

```diff
@@ -11,10 +11,10 @@ before_script:
 - haxelib git debugger https://github.com/TiVo/debugger.git
 - mkdir build
 - mkdir report
-script: make test
+script: make ANT_TARGET=$ANT_TARGET test
 env:
-    - IDEA_VERSION=13.1.6
-    - IDEA_VERSION=14.0.4
-    - IDEA_VERSION=14.1.6
+    - IDEA_VERSION=14.0.4 ANT_TARGET=test
+    - IDEA_VERSION=14.1.6 ANT_TARGET=test
+    - IDEA_VERSION=15.0.3 ANT_TARGET=compile_test
```

Comment:
I added the target so that we won't try to test the v15 version until JetBrains fixes their code.  We still do the compile, though to make sure the sources are clean.